_tests/browser_tests/spec_suites/shared/shared_careers-students-and-graduates.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var StudentsAndGraduates = require( '../../page_objects/page_careers-students-and-graduates.js' );
+
+describe( 'Beta: Careers/Student-and-graduates', function() {
```

Comment:
I'm matching what's on here: https://github.com/cfpb/cfgov-refresh/blob/flapjack/_tests/browser_tests/spec_suites/shared/shared_contact-us.js#L3

But I agree. Let's remove the "Beta." Mind if I grab all those in another PR?